templates/annotations-dp.json

```diff
@@ -0,0 +1,7 @@
+{
+  "metadata": {
+    "annotations": {
+    	"meshery/component-type":"control-plane"
```

Comment:
I am not sure if this was supposed to be `"meshery/component-type": "data-plane"`?

Reply:
Oh yes data-plane, damn!